blues/__init__.py

```diff
@@ -1,5 +0,0 @@
-from blues.ncmc import *
```

Comment:
Doesn't there need to be an `__init__.py`? You must be doing something here that's beyond my understanding of python packaging, @nathanmlim ...